frontend/src/RecordingPageComp/RecordingPage.jsx

```diff
@@ -24,59 +27,70 @@ function RecordingPage() {
         setIsMidiReady(true);
       }
     };
-    setTimeout(checkMidiInitialization, 1000); 
-  }, []);
+    setTimeout(checkMidiInitialization, 1000);
+  }, [selectedBlob]);
 
   return (
-    <div className="main">
-      <NavBar />
-      <footer>
-        <MIDISounds
-          ref={midiSounds}
-          className="MIDIsoundsLogo"
-          appElementName="root"
-          instruments={[3]}
-        />
-      </footer>
-      <div className="user-info">
-        {user ? (
-          <>
-            <span> {user.username} Session.</span>
-          </>
-        ) : (
-          <Link to="/">Error: Please go Back</Link>
-        )}
-      </div>
+    <>
+      <div className="main">
+        <NavBar />
+        <div className="user-info">
+          {user ? (
+            <>
+              <span> {user.username} Session.</span>
+            </>
+          ) : (
+            <Link to="/">Error: Please go Back</Link>
+          )}
+        </div>
 
-      <div className="recording-container">
         <RecordingComp
           setNote={setNote}
           setPlaybackDuration={setPlaybackDuration}
+          selectedBlob={selectedBlob}
```

Comment:
The props don't seem to be defined or used in the RecordingComp component. Should these be removed?